test/types/optionSelect.test.tsx

```diff
@@ -58,4 +61,32 @@ describe('optionSelect', () => {
     expect(tester.text()).not.toContain('Yes');
     expect(tester.text()).not.toContain('2 Values');
   });
+
+  it('Shows search for large data sets, or on command', async () => {
+    // tslint:disable no-magic-numbers
+    const states = Array(50).map((_v: any) => ({
+        name: fakeTextShort(),
+        value: faker.random.uuid(),
+      }))
+      , fieldConfig = { field, options: states, optionType: 'state', type };
+
+    function getPropsFor (overrides: Partial<IFieldConfig> = {}) {
+      return { props: { fieldSets: [[{ ...fieldConfig, ...overrides }]] } };
+    }
+
+    function isSearchable (tester: Tester) {
+      return !!tester.find('input').length;
+    }
+
+    async function expectIsSearchable (overrides: Partial<IFieldConfig>, tobe: boolean)  {
```

Comment:
I think `tobe` would be clearer as `expectedValue`. If you disagree heavily, then you can keep it but please camel case